.github/workflows/pr-acceptance-ce.yml

```
@@ -0,0 +1,52 @@
+# This workflow runs acceptance tests on pull requests (CE)
+
+name: pr-acceptance-ce
+
+on:
+  pull_request:
+    # Acceptance tests are unnecessary to run on some types of PRs.
+    paths-ignore:
+      - 'docs/**'
+      - 'examples/**'
+      - 'README.md'
+      - 'CHANGELOG.md'
+      - 'CONTRIBUTING.md'
+
+concurrency: 
+  group: ${{ github.workflow }}-${{ github.event.pull_request.number }}
+  cancel-in-progress: true
+
+jobs:
+  go-version:
+    runs-on: ubuntu-latest
+    outputs:
+      go-version: ${{ steps.go-version.outputs.go-version }}
+    steps:
+      # Check out the pull request code (as opposed to the target project).
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
```

We can use the default settings here now.

```suggestion
      - uses: actions/checkout@v2
```